loaders/index.js

```diff
@@ -0,0 +1,9 @@
+import expressLoader from './express';
+import sequelizeLoader from './sequelize';
+
+export default async ({ expressApp }) => {
+  await sequelizeLoader();
+  console.log('Sequelize Initialized');
+  await expressLoader({ app: expressApp });
+  console.log('Express Initialized');
+};
```

Comment:
You missed an implementation for:
"Write SQL script which will create Users table in the DB and fill it in with predefined users’ collection."

Could have been included in the loaders.